swift/ios-application/build.gradle

```diff
@@ -0,0 +1,26 @@
+buildscript {
+    dependencies {
+        classpath "org.gradle.samples:ios-plugin:latest.integration"
+    }
+}
+
+plugins {
+    id 'swift-application'
+    id 'xcode'
+}
+
+apply plugin: 'org.gradle.samples.ios-application'
+
+group = "org.gradle.samples"
+
+// Add iOS specific resource to Xcode IDE project
```

Comment:
The following stuff looks like it belongs in the sample plugin. It doesn't seem to make sense that every project in every build should want or need to configure all this stuff and that the plugin can instead provide a convention.